libs/convert/job.go

```diff
@@ -0,0 +1,35 @@
+package convert
+
+import (
+	"github.com/databricks/cli/libs/config"
+	"github.com/databricks/databricks-sdk-go/service/jobs"
+)
+
+var taskFieldOrder = []string{"TaskKey", "DependsOn", "ExistingClusterId", "NewCluster", "JobClusterKey"}
+var jobFieldOrder = []string{"Name", "Format", "NewCluster", "JobClusters", "ExistingClusterId", "Compute", "Tasks"}
```

Comment:
We should mask out defaults. For example, the `format` is always `MULTI_TASK` given we're using the 2.1 API. I suspect there are other defaults that we could/should omit as well for brevity.